### ISSUE TITLE
Problem with audio resume

### ISSUE THREAD

**Issue by gb160** [alexa/alexa-avs-sample-app#1015]
Hi, slight issue im hoping someone can help me out with, running the sample app on a Pi2.
I use tune in radio and often have it playing through Alexa, but if I interrupt it and call Alexa during playback, like ask her the time, she responds with the time ok, but when tune in radio resumes it will nearly always stutter for about a minute or so, before playing back correctly....ive tried with both wifi and ethernet and its the same...the workaround is to call Alexa again, stop tunein, then start it again with another request to Alexa, when the stuttering is happening, ive checked the system load via top and theres no excessive load, cpu is about 80% idle.

another slight issue which is kind of similar is again when using tune in, and I interrupt playback with a request, I get no sound to signify Alexa is listening, just silence, then after my request I get the both the listening/end listening sounds straight after each other.

**Comment by priyagsenthil** [comment]
Hi gb160,

We have noticed this issue with pi3. pi2 is known to be a little slower than pi3. We will try to try on a pi2. Also which version of the SDK are you using. Things are significantly better with SDKv1.2.1. Could you try with this version and provide an update while we investigate the issue with pi2.

Thanks
Priya

**Comment by gb160** [comment]
@priyagsenthil Im using the java sample app, but due to the fact id messed with so many different configuration/settings etc I decided to reinstall Raspbian Jessie from scratch, I then reinstalled Alexa-AVS and it all seems to be working again, no buffering, I can interrupt playback with a request and after answering the request Alexa resumes playback fine now...its also solved the issue where I wasn't getting the listening sound.
Ill close this issue and reopen it if It reoccurs.
Thanks.
